backend/resources/projects.py

```diff
@@ -118,63 +120,61 @@ def projects_post():
 
     # ToDo: Implement POST /projects
 
-    authToken = request.headers.get('authToken', default=None)
+    auth_token = request.headers.get('authToken', default=None)
     name = request.headers.get('name', default=None)
-    webpage = request.headers.get('webpage', default="")
-    idInstitution = request.headers.get('idInstitution', default="")
+    webpage = request.headers.get('webpage', default=None)
+    id_institution = request.headers.get('idInstitution', default=None)
```

Comment:
Sicher, dass der Default wert auf None sein sollte?

Wenn ja, müssen da die noch initialisiert werden, sollten die nicht gesetzt sein, da die ansonsten als None in die .add() Funktion der Session geschrieben werden. Weiß aber auch nicht, ob SQLAlchemy schlau genug ist den DB Insert an der Stelle dann mit NULL zu machen. Dann soll's mir recht sein.

Reply:
ist schlau genug